Header header

string host

float64 load_avg_1min
float64 load_avg_5min
float64 load_avg_15min

int64 mem_bytes_total
int64 mem_bytes_used

float32 cpu_temperature

bool has_gpu
int64 gpu_mem_bytes_total
int64 gpu_mem_bytes_free
float64 gpu_load
float32 gpu_temperature

int64 net_rx_bytes_sec
int64 net_tx_bytes_sec
